src/Toolkit/Toolkit.WPF/UI/Controls/FloorFilter/FloorFilter.Theme.xaml

```diff
@@ -0,0 +1,826 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="clr-namespace:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    xmlns:converter="clr-namespace:Esri.ArcGISRuntime.Toolkit.Internal"
+    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+    <converter:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
+    <system:Double x:Key="CollapsedWidth">48</system:Double>
+
+    <Color x:Key="DefaultBackgroundColor">White</Color>
```

Comment:
All key names in this file are too generic. If they are specific to floorfilter, make the key names floor filter specific.